go/tao/linux_docker_container_factory.go

```diff
@@ -61,35 +67,88 @@ func (dc *DockerContainer) WaitChan() <-chan bool {
 
 // Kill sends a SIGKILL signal to a docker container.
 func (dc *DockerContainer) Kill() error {
-	c := exec.Command("docker", "kill", dc.ContainerName)
-	return c.Run()
+	cid, err := dc.ContainerName()
+	if err != nil {
+		return err
+	}
+	return docker(nil, "kill", cid)
+}
+
+func (dc *DockerContainer) ContainerName() (string, error) {
+	b, err := ioutil.ReadFile(dc.CidfilePath)
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSpace(string(b)), nil
+}
+
+func docker(stdin io.Reader, args ...string) error {
```

Comment:
I think it will be useful in the future to use the docker client interface here, or at least to use the socket and the JSON format. In that case, it will be useful to know what the command is that we're sending. So, let's call that out in an explicit parameter here